python_modules/dagster/dagster/core/definitions/decorators/job.py

```diff
@@ -144,6 +148,9 @@ def job(
         version_strategy (Optional[VersionStrategy]):
             Defines how each op (and optionally, resource) in the job can be versioned. If
             provided, memoizaton will be enabled for this job.
+        partitions_def (Optional[PartitionsDefinition]): Defines a screet set of partition keys
```

Comment:
is this supposed to be `discrete`?  Or does `screet` mean something I'm not aware of...

Reply:
Oops, definitely a typo.